### PR TITLE
[Balance] Unfettered Ion/Scrambling tweaks

### PR DESCRIPTION
**Pull request by EjoThims**
## Summary
Switches the new Unfettered Ionic Prototype weapons to not have mirrored Ion and Scramble values.

Total values for both have been slightly lowered (.05 for Blaster and .5 for Turret). Both maintain a roughly 3:1 ratio (50% increase to one and 50% reduction to other).

This is being PRd as draft in order to get these numbers reviewed and refined at least somewhat from here before being declared ready.

### CODE REVIEW

File: data/hai/hai outfits.txt [hunk 1]
```diff
@@ -188,9 +188,9 @@ outfit "Ionic Turret Prototype"
 		"hit force" 60
 		"shield damage" 100
 		"hull damage" 50
-		"ion damage" 14
-		"scrambling damage" 14
-	description "This weapon improves upon the ionization capabilities of the Ion Cannon, while also being turret-mounted, enabling it to fire at enemies in any direction, not just those directly ahead."
+		"ion damage" 22
```
Comment (Hurleveur) [inline data/hai/hai outfits.txt:191]: I dont mind this but its crazily too good, just re-use the values from the campaign, they feel fair (14 ion and 10 scrambling, I think)

**Comment by Hurleveur** [comment]
Thx! The blaster makes little sense without these tweaks, and it's good to split said tweaks from the rest of the campaign indeed.

File: data/hai/hai outfits.txt (hunk 1 above)
Comment (EjoThims) [reply data/hai/hai outfits.txt:191]:
```suggestion
		"ion damage" 17.5
```

File: data/hai/hai outfits.txt [hunk 2]
```diff
@@ -188,9 +188,9 @@ outfit "Ionic Turret Prototype"
...
+		"scrambling damage" 4
```
Comment (EjoThims) [inline data/hai/hai outfits.txt:192]:
```suggestion
		"scrambling damage" 10
```

**Review by Hurleveur — APPROVED** [review]
Looks good to me for sure!
The turrets are intentionally nerfed more because an ion damage increase is more valuable than scrambling, I think. And this way they're not too strong (this is how it is in UHai campaign, value which was playtested and adjusted quite a bit)

**Pull request merged**